### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by PandelisZ**

### CODE REVIEW

**Review by mattburman — CHANGES_REQUESTED** [review]
hmm, i cant quite put my finger on it, but i think this PR needs work

File: README.md [hunk 1]
```diff
@@ -1 +1 @@
-# Matt Burman
+# Matt Burnman
```
Comment (sprusr) [inline README.md]:
```suggestion
# Matt Birdman

![birdman](https://user-images.githubusercontent.com/7497084/87015159-b493c780-c1d5-11ea-9bac-c281e1dc272a.png)
```

**Review by sprusr — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (mattburman) [reply README.md]: close but no cigar

**Comment by PandelisZ** [comment]
I've implemented the feedback and suggestions

**Review by sprusr — APPROVED** [review]
LGTM

**Review by mattburman — CHANGES_REQUESTED** [review]
whilst I appreciate the intent behind 6b801ba and deem it a valuable contribution, I feel it removes a certain essence of the project, thus I must request a minor modification, specifically around the middle columns of this diff

@PandelisZ

**Comment by mattburman** [comment]
_does it say burning_? 😆

**Comment by mattburman** [comment]
Closed due to inactivity. If this is still an issue feel free to reopen. Thanks!

**Pull request closed without merging**